plugins/modules/net_tools/ldap/ldap_attr.py

```diff
@@ -255,11 +257,28 @@ def main():
         module.fail_json(msg=missing_required_lib('python-ldap'),
                          exception=LDAP_IMP_ERR)
 
-    # Update module parameters with user's parameters if defined
-    if 'params' in module.params and isinstance(module.params['params'], dict):
+    # For Ansible-2.9.x and below, allow the params module parameter with a warning
+    if LooseVersion(module.ansible_version) < LooseVersion('2.10'):
```

Comment:
I'm not sure whether this still makes sense. I would rather have this in the collection behave as "2.10 or newer", and the backport to stable-2.9 cover the other case. The original `if` made more sense while this code was still in ansible/ansible.

Reply:
(If it makes backporting easier, we could merge this first and then update it to remove that `if`.)

Reply:
Agreed, I'll just update the backport to include this. Fixed in 95ca4ea08dc251f21aea11086f4149a15464858e.